src/tasks.py

```diff
@@ -176,20 +176,21 @@ def update(ctx):
 def migrations(ctx):
     print("**************************migrations*******************************")
     ctx.run(f"python manage.py migrate --noinput --settings={_localsettings()}", pty=True)
-    print("**************************datastore*******************************")
-    ctx.run(f"python manage.py migrate --noinput --database datastore --settings={_localsettings()}", pty=True)
```

Comment:
this was needed for the dynamic model schema creation. The migration must be apply on both databases

Reply:
I see, my fault, I'll revert this one, thanks

Reply:
Reverted